packages/cogify/src/cogify/cli/cli.cover.ts

```diff
@@ -76,6 +76,14 @@ export const BasemapsCogifyCoverCommand = command({
 
     const res = await createTileCover(ctx);
 
+    // Find the dem/dsm prefix for the nz-elevation bucket source and update imagery name to include prefix
+    if (im.collection != null) {
+      const geoCategory = im.collection['linz:geospatial_category'];
+      if (geoCategory === 'dem' || geoCategory === 'dsm') {
+        im.name = `${im.name}_${geoCategory}_${im.gsd}m`;
```

Comment:
`im.gsd` is a floating point number so you may see something like `1.00004` it would be good to trim it to `1m` or `0.1m` `0.75m`